src/Take.Blip.Builder/Content/MessageExtensions.cs

```diff
@@ -0,0 +1,30 @@
+﻿using Lime.Protocol;
+using Take.Blip.Client.Content;
+
+namespace Take.Blip.Builder.Content
+{
+    public static class MessageExtensions
```

Comment:
Creating a extension class to manage input expiration may be a overuse.
I don't think this methods are rich for the message object, but for the cases which input will have a expiration.
Maybe a helper class can be a better approach. What do you think @andreminelli @andrebires ?